pymc/distributions/logprob.py

```diff
@@ -320,3 +324,51 @@ def ignore_logprob(rv: TensorVariable) -> TensorVariable:
         return rv
     new_node = assign_custom_measurable_outputs(node, type_prefix=prefix)
     return new_node.outputs[node.outputs.index(rv)]
+
+
+@_logprob.register(MeasurableJoin)
```

Comment:
Do we expect to have more of these AePPL logprob dispatches to be overwritten? Because symbolic distributions don't have a `logp` method, I was getting confused on how the log-probabilities are being retrieved

Reply:
We should try to avoid as much duplicated work as possible, so I imagine it won't be that frequent.

What do you mean by symbolic distributions don't have logp methods? All except Censored right now have direct logp methods.

Reply:
Sorry, perhaps that wasn't clear or my terminology was wrong. By `logp` method, I meant that they are defined underneath the Distribution class rather than assigned via single dispatch. The second sentence was more of a comment/concern, maybe it doesn't make too much sense...

Reply:
That is a shortcut we did sometime ago where the MetaDistribution will do the dispatching automatically. Some context in this discussion: https://github.com/pymc-devs/pymc/issues/5308